resources/views/hardware/checkout.blade.php

```diff
@@ -161,6 +161,26 @@
                             </div>
                         </div>
 
+
+                        @can("audit",$asset)
+                        <!-- Log an audit checkbox -->
+                        <div class="form-group">
+                            <div class="col-sm-3 control-label" >
+                                <label>
+                                    {{ trans('admin/settings/general.audit_on_checkinout') }}
+                                </label>
+                            </div>
+                            <div class="col-md-8">
+                                <label class="form-control">
+                                    <input type="checkbox" value="1" name="audit_on_checkinout" {{ (old('audit_on_checkinout')) == '1' ? ' checked="checked"' : '' }} aria-label="audit_on_checkinout">
```

Comment:
I think I'd prefer the value be called `log_audit`, and the GUI should show nothing on the left side, with just the text "Mark as audited".